plugins/salt/_salt

```diff
@@ -145,7 +145,7 @@ _master_options=(
     '(-v --verbose)'{-v,--verbose}'[Turn on command verbosity, display jid and active job queries]'
     '--hide-timeout[Hide minions that timeout]'
     '(-b --batch --batch-size)'{-b,--batch,--batch-size}'[Execute the salt job in batch mode, pass number or percentage to batch.]:Batch Size:'
-    '(-a --auth --eauth --extrenal-auth)'{-a,--auth,--eauth,--external-auth}'[Specify an external authentication system to use.]:eauth:'
+    '(-a --auth --eauth --external-auth)'{-a,--auth,--eauth,--external-auth}'[Specify an external authentication system to use.]:eauth:'
```

Comment:
Based on the preceding line, this seems likely to be correct, but I can't state with any confidence what this changes.